domain-cc/cc-app/src/python_src/api.py

```diff
@@ -82,6 +82,31 @@ def log_as_json(log: dict):
     logging.info(json.dumps(log))
 
 
+def log_claim_stats(claim: Claim, classification: Optional[PredictedClassification]):
+    if classification:
+        classification_code = classification.classification_code
+        classification_name = classification.classification_name
+    else:
+        classification_code = None
+        classification_name = None
+
+    contention_text = claim.contention_text or ''
+    is_in_dropdown = contention_text.strip().lower() in dropdown_values
+    sanitized_contention_text = contention_text if is_in_dropdown else "Not in dropdown"
+
+    log_as_json({
+        "claim_id": sanitize_log(claim.claim_id),
+        "claim_type": sanitize_log(claim.claim_type),
+        "classification_code": classification_code,
+        "classification_name": classification_name,
+        "contention_text": sanitized_contention_text,
+        "diagnostic_code": sanitize_log(claim.diagnostic_code),
+        "form526_submission_id": sanitize_log(claim.form526_submission_id),
+        "is_in_dropdown": is_in_dropdown,
+        "is_lookup_table_match": classification_code is not None,
```

Comment:
I think we should add the contention text if it was in a lookup table even if it is not in the current dropdown option. in our current logs we use the already_mapped_text field to do this.